nltk/translate/bleu_score.py

```diff
@@ -150,8 +150,8 @@ def corpus_bleu(
     :type list_of_references: list(list(list(str)))
     :param hypotheses: a list of hypothesis sentences
     :type hypotheses: list(list(str))
-    :param weights: weights for unigrams, bigrams, trigrams and so on
-    :type weights: list(float)
+    :param weights: weights for unigrams, bigrams, trigrams and so on, (one or list of weights)
+    :type weights: tuple(float) or list(tuple(float))
```

Comment:
```suggestion
    :type weights: tuple(float) / list(tuple(float))
```
I believe the proper way (In Sphinx default documentation) to specify that a parameter may be one of multiple types is using a `/`. Perhaps in the future we can rework all of this using the Python 3.5+ type hinting, or the Python 3 `typing`'s `Union[Tuple[float], List[Tuple[float]]` - but this is something for later, for the entire codebase (or large sections) at once.

Also see https://github.com/nltk/nltk/blob/f989fe65d421e7ea4d1037a00f07eaeee3ad6a29/nltk/translate/chrf_score.py#L77 for an example of this.

Reply:
Fixed in c730dc3. I'm open to suggestions or changes.